src/openrct2-unity/Assets/Scripts/UI/PeepInformation.cs

```diff
@@ -13,7 +13,6 @@ public class PeepInformation : MonoBehaviour
     int nausea;
     int toilet;
     [SerializeField] private GameObject peepBox;
-    [SerializeField] private GameObject referenceObject;
     GameObject PeepBox;
```

Comment:
Two variables with the same name? If they're different, their names should clearly show the difference. :)